models/destination_policies.go

```diff
@@ -0,0 +1,30 @@
+package models
```

Comment:
In model folders, all the file names are named in singular. I tend to think that a model is an abstraction of a single item, so that why I see that is a singular name.

Reply:
I agree, but I saw you started the plural with "route_rules.go" one and I just followed it :-).
It's not a problem I can move those to singular, I tend to name in plural as well, and everyday Jay makes me change it.
Let me know if that refactoring make sense on both files.

Reply:
Upsie... I knew that something would explote in my face, haha.
I think this is super opinionated question. IMO I would change it to singular, but that is my way of thinking. And I will refactor de route_rules.go to route_rule.go.

Reply:
Don't worry, I can do both in this PR.
It's a minor change.